.github/workflows/test.yaml

```diff
@@ -451,16 +453,23 @@ jobs:
           # Register a problem matcher to highlight spread failures
           echo "::add-matcher::.github/spread-problem-matcher.json"
 
+          BACKEND=google
+          SPREAD=spread
+          if [[ "${{ matrix.system }}" =~ -arm- ]]; then
+              BACKEND=google-arm
+              SPREAD=spread-arm
```

Comment:
It's not clear why we need a different spread command (and where it is).

Reply:
well, this is something temporal until we have the change merged in spread, so I'll remove spread-arm